synapsemonitor/monitor.py

```diff
@@ -100,7 +102,14 @@ def _find_modified_entities_file(syn: Synapse, syn_id: str, days: int = 1) -> li
     Returns:
         List of synapse ids
     """
-    raise NotImplementedError("Files not supported yet")
```

Comment:
I wonder if this function should be called `_find_modified_entities_single_entity`?